### PR TITLE
add llms mbti code, streamlit app, readme

### PR DESCRIPTION
**Pull request by Derekwang2002**
date: 7.5

### CODE REVIEW

File: README.md
Comment (yihong-eric-chen) [inline README.md:1]: Add instructions and scripts to set up the environment for this project.

File: gpt-api-starter.ipynb
Comment (yihong-eric-chen) [inline gpt-api-starter.ipynb:1]: Let users know how to create '.env.\n' and where to store the key in README

File: README.md
Comment (yihong-eric-chen) [inline README.md:1]: Order to execute notebooks and python scripts matters! 'gpt-api-starter.ipynb' + 'mbti_questions.json' -> 'llms_mbti.json', then 'llms_mbti.json' can be further used in 'web.py'. Let users know the structure of the project and how to execute each script in order.

File: web.py
Comment (yihong-eric-chen) [inline web.py:1]: Remove '\n', there can be 2 separate columns to display 'Question' and 'Answers'.
image

**Comment by 118020020** [comment]
1. You may try to package the images in one folder for a clearer structure. You can refer to [this](https://backrightup.com/blog/how-to-organize-github-repositories/) for more details. But you do not need to be so complicated since it's a rather easy project. But ideally, it is better to let viewers know the function of each file once they see the repo.
2. For more about creating a nice readme file, you can refer to this [link](https://www.freecodecamp.org/news/how-to-write-a-good-readme-file/).

**Comment by 118020020** [comment]
> date: 7.5

1. You may try to package the images in one folder for a clearer structure and consider using some tricks to better organize the repo. You can refer to [this](https://backrightup.com/blog/how-to-organize-github-repositories/) for more details. But you do not need to be so complicated since it's a rather easy project. Ideally, it is better to let viewers know the function of each file once they see the repo.
2. For more about creating a nice readme file, you can refer to this [link](https://www.freecodecamp.org/news/how-to-write-a-good-readme-file/).

**Comment by CH-chuan** [comment]
1. You may try to package the images in one folder for a clearer structure and consider using some tricks to better organize the repo. You can refer to [this](https://backrightup.com/blog/how-to-organize-github-repositories/) for more details. But you do not need to be so complicated since it's a rather easy project. Ideally, it is better to let viewers know the function of each file once they see the repo.
2. For more about creating a nice readme file, you can refer to this [link](https://www.freecodecamp.org/news/how-to-write-a-good-readme-file/).

**Pull request merged**